ui/components/component-library/picker-network/picker-network.scss

```diff
@@ -0,0 +1,6 @@
+.mm-picker-network {
+  --picker-network-height: 32px;
+
+  max-width: 100%;
+  height: var(--picker-network-height);
+}
```

Comment:
I think we need to include import in `ui/components/component-library/component-library-components.scss` then the height css should work